### PR TITLE
Edit pitr compatible description with add index lightning feature

### PR DESCRIPTION
**Pull request by Benjamin2037**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add description of compatiable problem between PITR with Add index lightning way

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -679,6 +679,9 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
 
+> **注意：**
+> 当前索引加速功能与 PITR 功能不兼容，用户在需要使用索引加速功能的时候，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。
```
Comment (Oreoxmt) [inline system-variables.md:683]:
```suggestion
> **警告：**
>
> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。
```

非预期结果可以简单说明可能会出现哪些“非预期结果”吗

Comment (Benjamin2037) [reply system-variables.md:683]: 我们做了兼容性判断，这里非预期效果也其实就是一个 corner case，用户同时启动 pitr 和 索引加速。可能会导致两个同时启动，这个时候 PITR 备份的索引会有问题。不过这个问题不常见，不需要明确说吧？

File: system-variables.md [hunk 2]
```diff
@@ -679,6 +679,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
 
+> **警告：**
+>
+> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。
+> 非预期情况如下：
+> 如果先启动 PITR 任务，增加索引不会走快速添加模式，索引加速效果下降，不影响索引正确性。
+> 如果先启动快速添加索引任务，再启动 PITR 任务会报错，启动 PITR 任务失败，正在添加索引的任务没有影响。
+> 如果同时启动 PITR 任务和快速添加索引任务，会出现添加的索引，可能没有包括全量数据在此次 PiTR 备份中。
```
Comment (Oreoxmt) [inline system-variables.md:688]:
```suggestion
> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
> - 如果先启动 PITR 备份任务，再添加索引，此时索引不会加速添加，但不影响索引正确性。
> - 如果先启动添加索引加速任务，再启动 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
> - 如果同时启动 PITR 备份任务和添加索引加速任务，此时 PITR 备份数据可能不完整。
```

File: br/pitr-known-issues.md [hunk 3]
```diff
@@ -56,3 +56,12 @@ Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
...
 当场景中有大事务时，日志 checkpoint lag 在事务提交前都不会更新，因此会增加一段接近于大事务提交时长的时间。
+
+## 当集群正在使用索引加速功能时,创建日志备份任务会失败
+
+Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
+
+当前[索引加速功能](/system-variables.md#tidbddlenablefastreorg-span-class"version-mark"从-v630-版本开始引入span)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+- 如果先启动 PITR 备份任务，再添加索引，此时索引不会加速添加，但不影响索引正确性。
+- 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
+- 如果同时启动 PITR 备份任务和添加索引加速任务，此时 PITR 备份数据可能不完整，无法用于 PITR 数据恢复。
```
Comment (IANTHEREAL) [inline br/pitr-known-issues.md:67]:
```suggestion
- 如果同时启动 PITR 备份任务和添加索引加速任务，可能两个任务无法察觉到对方而造成 PITR 不能备份下来新增加的索引数据。
```

Comment (IANTHEREAL) [inline br/pitr-known-issues.md:67]:
```suggestion
- 如果同时启动 PITR 备份任务和添加索引加速任务，可能两个任务无法察觉到对方而造成 PITR 不能备份下来新增加的索引数据。
```

Comment (IANTHEREAL) [reply br/pitr-known-issues.md:67]: 帮忙添加到 pitr 限制里面 @joccau

File: br/pitr-known-issues.md [hunk 4]
```diff
@@ -56,3 +56,12 @@ Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
 Issue 链接：[#13304](https://github.com/tikv/tikv/issues/13304)
 
 当场景中有大事务时，日志 checkpoint lag 在事务提交前都不会更新，因此会增加一段接近于大事务提交时长的时间。
+
+## 当集群正在使用索引加速功能时,创建日志备份任务会失败
+
+Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
+
+当前[索引加速功能](/system-variables.md#tidbddlenablefastreorg-span-class"version-mark"从-v630-版本开始引入span)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+- 如果先启动 PITR 备份任务，再添加索引，此时索引不会加速添加，但不影响索引正确性。
```
Comment (seiya-annie) [inline br/pitr-known-issues.md:65]: 如果先启动 PITR 备份任务，再添加索引，此时及时索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。

**Review by seiya-annie — APPROVED** [review]
LGTM

File: br/pitr-known-issues.md [hunk 5]
```diff
@@ -56,3 +56,12 @@ Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
 Issue 链接：[#13304](https://github.com/tikv/tikv/issues/13304)
 
 当场景中有大事务时，日志 checkpoint lag 在事务提交前都不会更新，因此会增加一段接近于大事务提交时长的时间。
+
+## 当集群正在使用索引加速功能时,创建日志备份任务会失败
+
+Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
+
+当前[索引加速功能](/system-variables.md#tidbddlenablefastreorg-span-class"version-mark"从-v630-版本开始引入span)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
```
Comment (Oreoxmt) [inline br/pitr-known-issues.md:64]:
```suggestion
当前[索引加速功能](/system-variables.md#tidbddlenablefastreorg-span-class"version-mark"从-v630-版本开始引入span)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：

```

File: system-variables.md [hunk 6]
```diff
@@ -679,6 +679,13 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
 
+> **警告：**
+>
+> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
```
Comment (Oreoxmt) [inline system-variables.md:684]:
```suggestion
> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：

```

File: br/pitr-known-issues.md (hunk 4 above)
Comment (Oreoxmt) [inline br/pitr-known-issues.md:65]:
```suggestion
- 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
```

File: system-variables.md [hunk 7]
```diff
@@ -679,6 +679,13 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+> - 如果先启动 PITR 备份任务，再添加索引，此时索引不会加速添加，但不影响索引正确性。
+> - 如果先启动添加索引加速任务，再启动 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
+> - 如果同时启动 PITR 备份任务和添加索引加速任务，此时 PITR 备份数据可能不完整。
```
Comment (Oreoxmt) [inline system-variables.md:687]:
```suggestion
> - 如果同时启动 PITR 备份任务和添加索引加速任务，可能两个任务无法察觉到对方而造成 PITR 不能备份下来新增加的索引数据。
```

File: system-variables.md [hunk 8]
```diff
@@ -679,6 +679,13 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
 
+> **警告：**
+>
+> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+> - 如果先启动 PITR 备份任务，再添加索引，此时索引不会加速添加，但不影响索引正确性。
```
Comment (Oreoxmt) [inline system-variables.md:685]:
```suggestion
> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
```

File: br/pitr-known-issues.md [hunk 9]
```diff
@@ -56,3 +56,13 @@ Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
 Issue 链接：[#13304](https://github.com/tikv/tikv/issues/13304)
 
 当场景中有大事务时，日志 checkpoint lag 在事务提交前都不会更新，因此会增加一段接近于大事务提交时长的时间。
+
+## 当集群正在使用索引加速功能时，创建日志备份任务会失败
+
+Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
+
+当前[索引加速功能](/system-variables.md#tidbddlenablefastreorg-span-class"version-mark"从-v630-版本开始引入span)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
```
Comment (Oreoxmt) [inline br/pitr-known-issues.md:64]:
```suggestion
当前[索引加速功能](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
```

File: system-variables.md [hunk 10]
```diff
@@ -679,6 +679,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
 
+> **警告：**
+>
+> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+
```
Comment (Oreoxmt) [inline system-variables.md:685]:
```suggestion
>
```

File: system-variables.md [hunk 11]
```diff
@@ -679,6 +679,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
 
+> **警告：**
+>
+> 当前索引加速功能与 [PITR (Point-in-time recovery)](/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
```
Comment (Oreoxmt) [inline system-variables.md:684]:
```suggestion
> 当前索引加速功能与 [PITR (Point-in-time recovery)](/br/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
```

File: br/pitr-known-issues.md [hunk 12]
```diff
@@ -56,3 +56,13 @@ Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
...
+
+## 当集群正在使用索引加速功能时，创建日志备份任务会失败
+
+Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
+
+当前[索引加速功能](/system-variables.md#tidb_ddl_enable_fast_reorg-从-v630-版本开始引入)与 PITR 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+
+- 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
+- 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
+- 如果同时启动 PITR 备份任务和添加索引加速任务，可能两个任务无法察觉到对方而造成 PITR 不能备份下来新增加的索引数据。
```
Comment (hfxsd) [inline br/pitr-known-issues.md:68]:
```suggestion
- 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
```

File: system-variables.md [hunk 13]
```diff
@@ -679,6 +679,14 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 - 默认值：`OFF`
 - 这个变量是用来控制是否开启添加索引加速功能，来提升创建索引回填过程的速度。如果本开关启动，TiDB 在进行创建索引任务时将会采用更加高效的方式完成索引创建。
 
+> **警告：**
+>
+> 当前索引加速功能与 [PITR (Point-in-time recovery)](/br/point-in-time-recovery.md) 功能不兼容。在使用索引加速功能时，需要确保后台没有启动 PITR 备份任务，否则可能会出现非预期结果。非预期场景包括：
+>
+> - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
+> - 如果先启动添加索引加速任务，再启动 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
+> - 如果同时启动 PITR 备份任务和添加索引加速任务，可能两个任务无法察觉到对方而造成 PITR 不能备份下来新增加的索引数据。
```
Comment (hfxsd) [inline system-variables.md:688]:
```suggestion
> - - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
```

File: br/pitr-known-issues.md [hunk 14]
```diff
@@ -56,3 +56,13 @@ Issue 链接：[#37207](https://github.com/pingcap/tidb/issues/37207)
 Issue 链接：[#13304](https://github.com/tikv/tikv/issues/13304)
 
 当场景中有大事务时，日志 checkpoint lag 在事务提交前都不会更新，因此会增加一段接近于大事务提交时长的时间。
+
+## 当集群正在使用索引加速功能时，创建日志备份任务会失败
```
Comment (hfxsd) [inline br/pitr-known-issues.md:60]: 索引加速功能与 PITR 功能不兼容

**Review by hfxsd — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**